tests/by-util/test_dd.rs

```diff
@@ -1186,3 +1186,16 @@ fn test_final_stats_si_iec() {
     let s = result.stderr_str();
     assert!(s.starts_with("2+0 records in\n2+0 records out\n1024 bytes (1 KB, 1024 B) copied,"));
 }
+
+#[test]
+fn test_partial_skip_stdin() {
```

Comment:
@sylvestre 

Well... I added the test, but it does not work as expected and fails with `Illegal seek`. As any other test which can try seek `stdin`.

However, it can work outside the test system (for example, if I run `tests/misc/head-c.sh`)

I am confused why it does not work in the test system. Let me know if there are any insights on this.